GP/toy_experiments/toy_sin_noise_with_rand.py

```diff
@@ -0,0 +1,201 @@
+# Copyright Lee Group 2019
```

Comment:
We can just do the random sampling in the original script rather than creating a new one.